steps/base.py

```diff
@@ -24,6 +38,95 @@ def __init__(self,
                  load_saved_output=False,
                  save_graph=False,
                  force_fitting=False):
+        """
+        Args:
+            name (int): Step name. Each step in a pipeline needs to have a unique name.
+                Transformers, and Step outputs will be persisted/cached/saved under this exact name.
+            transformer (obj): Step instance or object that inherits from BaseTransformer.
+                When Step instance is passed transformer from that Step will be copied and used to perform transformations.
+                It is useful when both train and valid data are passed in one pipeline (common situation in deep learnining).
+            input_steps (list): list of Step instances default []. Current step will combine outputs from input_steps and input_data
```

Comment:
I changed the default to None on one of the branches

Reply:
@grzes314 can I change it to None leaving everything else as is ?